src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Drawable {
+
```

Comment:
remove empty line after the class declaration, and do the same for other classes